src/audio/src_config.h

```diff
@@ -66,7 +66,11 @@
 #endif
 #else
 /* GCC */
+#if defined __x86_64__
```

Comment:
Probably better to conditionally select this if target is test bench ? otherwise testbench will use short coefficients if testbench is built on ARM.

Reply:
Could we define macro CONFIG_HOST for any host type build and check for it in SRC configuration?